src/passthrough/async_io.rs

```diff
@@ -314,6 +314,17 @@ impl<D: AsyncDrive + Sync, S: BitmapSlice + Send + Sync> AsyncFileSystem<D, S>
             })
             .await?;
 
+        let mut attr_flags: u32 = 0;
+        if let Some(dax_file_size) = self.cfg.dax_file_size {
+            // st.stat.st_size is i64
+            if self.perfile_dax.load(Ordering::Relaxed)
```

Comment:
Why uses two flags "self.cfg.dax_file_size" and "self.perfile_dax" to control the perfile-dax?

Reply:
Aha, got it. The `perfile_dax` reflects host/guest negotiating result.